app/src/vuex/modules/user.js

```diff
@@ -0,0 +1,86 @@
+import config from "src/../config"
```

Comment:
Did you consider the problems with mobile? https://firebase.google.com/docs/auth/web/email-link-auth#completing_sign-in_in_a_mobile_app